### ISSUE TITLE
UIAlertController forces orientation change to portrait, since 3.9.1

### ISSUE THREAD

**Issue by levochkaa** [cleveradssolutions/CAS-iOS#9]
After updating CAS version from 3.9.0 to 3.9.1 (we are using Optimal), all alerts started to force orientation change to portrait.

https://github.com/user-attachments/assets/b9242ff7-5455-42a1-b91c-f13719651d3e

And if the orientation is locked to landscape (what we have in production currently), the app crashes.
```
*** Terminating app due to uncaught exception 'UIApplicationInvalidInterfaceOrientation', reason: 'Supported orientations has no common orientation with the application, and [SwiftUI.PlatformAlertController shouldAutorotate] is returning YES'
```

It seems to get fixed after subclassing `UIAlertController` and adding a call to `setNeedsUpdateOfSupportedInterfaceOrientations`
```
override func viewDidLoad() {
    super.viewDidLoad()
    if #available(iOS 16, *) {
        setNeedsUpdateOfSupportedInterfaceOrientations()
    }
}
```
...but we have SwiftUI alerts..

After some investigation I've found, that the issue is probably in CASExchange adapter, because adding all Optimal adapters without it makes the bug disappear and vice-versa (adding only CASExchange brings the bug).

The same bug appears in CAS version 3.9.3

**Comment by Str4tos** [comment]
Hello @levochkaa,
Thank you for reporting this issue. We are currently investigating to determine the root cause. We appreciate your patience and will update you as soon as we have more information.

**Comment by Str4tos** [comment]
Thanks for your patience.

This issue will be fixed in our next release 3.9.4.
